examples/python_ocean_forced_simulation.jl

```diff
@@ -0,0 +1,125 @@
+# # A Python Ocean Simulation at 4ᵒ Resolution Forced by JRA55 Reanalysis and Initialized from ECCO
+#
+# This example showcases the use of ClimaOcean's PythonCall extension to run a
+# near-global ocean simulation at 4-degree resolution using the Veros ocean model.
+# The ocean is forced by the JRA55 reanalysis data and initialized from the ECCO
+# state estimate.
+#
+# For this example, we need Oceananigans, ClimaOcean, Dates, CUDA, and
+# CairoMakie to visualize the simulation.
+
+using ClimaOcean
+using PythonCall
+using Oceananigans
+using CairoMakie
+using Printf
+
+# We import the Veros 4 degree ocean simulation setup, which consists of a near-global ocean
+# with a uniform resolution of 4 degrees in both latitude and longitude and a latitude range spanning
+# from 80S to 80N. The setup is defined in the `veros.setups.global_4deg` module.
+
+# Before importing the setup, we need to ensure that the Veros module is installed and loaded
+# and that every output is removed to avoid conflicts.
+
+VerosModule = Base.get_extension(ClimaOcean, :ClimaOceanPythonCallExt)
+
+VerosModule.install_veros()
+VerosModule.remove_outputs(:global_4deg)
+
+# Actually loading and instantiating the Veros setup in the variable `ocean`.
+# This setup uses by default a different time-step for tracers and momentum, 
+# so we set it to the same value (1800 seconds) for both.
+
+ocean = VerosModule.VerosOceanSimulation("global_4deg", :GlobalFourDegreeSetup)
+
+set!(ocean, "dt_tracer", 1800.0; path=:settings)
+set!(ocean, "dt_mom",    1800.0; path=:settings)
+
+# We force the 4-degree setup with a prescribed atmosphere based on the JRA-55 reanalysis data.
+# This includes 2-meter wind velocity, temperature, humidity, downwelling longwave and shortwave
+# radiation, as well as freshwater fluxes.
+
+atmos = JRA55PrescribedAtmosphere(; backend = JRA55NetCDFBackend(10))
+
+
+# The coupled ocean--atmosphere model.
+# We use the default radiation model and we do not couple an ice model for simplicity.
+
+radiation = Radiation()
+coupled_model = OceanSeaIceModel(ocean, nothing; atmosphere=atmos, radiation)
+simulation = Simulation(coupled_model; Δt = 1800, stop_time = 60days)
+
+# We set up a progress callback that will print the current time, iteration, and maximum velocities
+# at every 5 iterations. It also collects the surface velocity fields and the net fluxes
+# into the arrays `s`, `tx`, and `ty` for later visualization.
+
+wall_time = Ref(time_ns())
+
+s  = []
+tx = []
+ty = []
```

Comment:
why can't you use a regular JLD2Writer? I think that would be preferred, to illustrate the user interface which is important.